app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -68,12 +67,7 @@ export const Body = () => {
     applications: config!.applications!,
     featureConfigs: config!.featureConfigs!,
     firefoxVersions: config!.firefoxVersions!,
-    // Populating owners from fetched experiments since it might be expensive
-    // to fetch and manage a list of all users
```

Comment:
Yeah, I guess this was kind of premature optimization. We could always lazy-load this later if it turns onerous to performance.

Reply:
Oh! I just realized this was the only use of `uniqueByProperty` so I should get rid of that too.